client/src/pages/Signup.js

```diff
@@ -0,0 +1,113 @@
+import { Box, Grid, Typography, TextField, Button } from '@material-ui/core'
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+import useStyles from '../styles'
```

Comment:
You should discuss with your team whether to keep useStyles with their corresponding component or in separate files, but it's generally a best practice to break up styles by component to keep imports small and components modular

Reply:
the reason I created a separate file was that most of the css property for signup & login pages are the same. I just wanted to not repeat the same css property in two different files. But I can break up the styles by component.